models/docs/sources/contract_data.md

```diff
@@ -22,3 +22,23 @@ Contract can either be temporary or persistent
 {% docs balance_holder %}
 The address/account that holds the balance of the asset in contract data
 {% enddocs %}
+
+{% docs key %}
+The encoded key used to identify a specific piece of contract data. This key is typically a hashed value representing the unique identifier for the data entry.
+{% enddocs %}
+
+{% docs key_decoded %}
+The human-readable or decoded version of the key. This provides an understandable format of the key, making it easier to interpret and use in analysis.
+{% enddocs %}
+
+{% docs val %}
```

Comment:
```The encoded val has two components (type and value) where type describes the data type and the value describes the encoded value of the data type for the contract data.```

This could be applied to `val` as well